src/main/kotlin/blackJack/domain/Card.kt

```diff
@@ -1,5 +1,7 @@
 package blackJack.domain
 
-data class Card(val shape: Shape, val denomination: Denomination) {
-    override fun toString() = "${shape.shape}${denomination.symbol}"
+data class Card(private val shape: Shape, private val denomination: Denomination) {
+    override fun toString(): String = "${shape.shape}${denomination.symbol}"
+
+    fun getNumber(): Int = denomination.number
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val number: Int = denomination.number
```